src/components/preloader/Pre.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PreloaderLottie from "../../assets/lottie/preLoader";
+import DisplayLottie from "../displayLottie/DisplayLottie";
+import {illustration} from "../../portfolio";
+import {useContext} from "react";
+import StyleContext from "../../contexts/StyleContext";
+import "./Pre.scss";
+
+function Pre(props) {
+  const {isDark} = useContext(StyleContext);
+
+  return (
+    <div
+      id={
+        isDark
+          ? props.load
+            ? "preloader-dark"
+            : "preloader-none"
+          : props.load
+          ? "preloader-light"
+          : "preloader-none"
+      }
+    >
+      {props.load ? (
+        illustration.animated ? (
+          <DisplayLottie animationData={PreloaderLottie} width={250} />
+        ) : (
+          <img
+            class="svg-loader "
+            alt="pre loader svg"
+            src={require("../../assets/pre.gif")}
```

Comment:
Hey @akshay-na, sorry for the delayed response.
Why are we using a gif here as a Lottie fallback? 

Also #366 has already implemented the splashscreen loader (and it was opened before your PR so it wouldn't be nice to reject that piece of work )

Hate to reject your first PR on GitHub 😅 but this is why it's better to ask on an issue before making a PR for it so we can expect it.
Maybe if @ashutosh1297 is alright with you picking up their PR's latest review you can help ship their branch. Or you can check out issue #456 and take a stab at it

Reply:
btw love your portfolio's loading animation ❤️